pkg/installmanager/installmanager.go

```diff
@@ -931,7 +876,7 @@ func initSSHAgent(privKeyPath string, m *InstallManager) (func(), error) {
 	return sshAgentCleanup, nil
 }
 
-func uploadInstallerLog(cd *hivev1.ClusterDeployment, m *InstallManager, installErr error) error {
+func uploadInstallerLog(provision *hivev1.ClusterProvision, m *InstallManager, installErr error) error {
```

Comment:
Per earlier comment this is probably the spot we need to add it to the provision and kill the configmap entirely.